extension_example/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+use dioxus::prelude::*;
+use extensions::*;
+use kit::{elements::button::Button, icons::Icon};
+
+// Exports the plugin using the registrar
+// You don't need to really worry about this but it is required.
+// Just change emoji's to the name of your extension in alpha-numeric snake case.
```

Comment:
"emojis" 
`// Just change "emojis"  to the name of your extension in alpha-numeric snake case.`